src/Analyzers/Core/CodeFixes/MatchFolderAndNamespace/AbstractChangeNamespaceToMatchFolderCodeFixProvider.CustomFixAllProvider.cs

```diff
@@ -72,7 +72,7 @@ static async Task<ImmutableArray<Diagnostic>> GetSolutionDiagnosticsAsync(FixAll
             private static async Task<Solution> FixAllByDocumentAsync(
                 Solution solution,
                 ImmutableArray<Diagnostic> diagnostics,
-                IProgressTracker progressTracker,
+                IProgress<CodeAnalysisProgress> progressTracker,
```

Comment:
Usafes of IProgressTracker  switch to `IProgress<T>` (A core .Net type) and a new Roslyn type `CodeAnalysisProgress` which provides information necessary to update the UI with the current state of hte long running operation.  This includes things like a 'description' of the current part of the op, as well as information about numeric progress, which can be used to update a progress bar.

Reply:
Note: to keep the diff small, any existing internal APIs keep the name `progressTracker`.  But in all new public APIs, the name is `progress`.  The next PR that goes in will rename *everything* to consistently be `progress`.

Reply:
Would it make sense to delete the IProgressTracker interface as part of this PR to make sure we've switched everything other than the names over?